fastembed/__init__.py

```diff
@@ -0,0 +1 @@
+from .embedding import TextEmbedding, DefaultEmbedding, FlagEmbedding, JinaEmbedding
```

Comment:
```suggestion
from .embedding import TextEmbedding, DefaultEmbedding, FlagEmbedding, JinaEmbedding

__all__ = [TextEmbedding, DefaultEmbedding, FlagEmbedding, JinaEmbedding]
```

Please run the formatter before pushing